integrations/pjrt/src/iree_pjrt/rocm/CMakeLists.txt

```diff
@@ -13,8 +13,8 @@ iree_cc_library(
     "client.cc"
   DEPS
     iree_pjrt::common
-    iree::experimental::rocm
-    iree::experimental::rocm::registration
+    iree::hal::drivers::hip
+    iree::hal::drivers::hip::registration
```

Comment:
Oof sorry that this code has bit-rotted so much, and thanks for finding what needed to be updated.